protos/v0/jvs_plugin_service.proto

```diff
@@ -42,3 +44,13 @@ message ValidateJustificationResponse {
   repeated string warning = 2;
   repeated string error = 3;
 }
+
+// GetUiDataRequest is to retrieve data for display. Currently, it is designed to obtain the display_name and hint. 
+message GetUiDataRequest {
```

Comment:
`Get` is considered a standard RPC method and cannot be dropped, I would suggest keep `Get`  to avoid confusion. See https://google.aip.dev/131 for more details

Reply:
Thank you for your suggestions. I agree with @dandans-verily and update the request name to `GetUIDataRequest`.

Reply:
Any chance you forgot to include the proto changes in the latest commit? I still see GetUiDataResponse

Reply:
Curious - which part of https://google.aip.dev/131 says we have to prefix it with `Get`?